### PR TITLE
Add wasmd scripts

### PR DESCRIPTION
**Pull request by orkunkl**
This PR adds wasmd setup to scripts, also makes `one-chain` script work with cosmos-sdk master with adding stake coin to `gentx`

### CODE REVIEW

File: scripts/build-wasmd [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+WASMDDIR="$(dirname $SCRIPTDIR)/chain-code/wasmd"
+
+[ ! -d "$WASMDDIR" ] && echo "Repository for akash does not exist at $WASMDDIR. Try running 'make get-akash'..." && exit 1
```
Comment (colin-axner) [inline scripts/build-wasmd]:
```suggestion
[ ! -d "$WASMDDIR" ] && echo "Repository for wasmd does not exist at $WASMDDIR. Try running 'make get-wasmd'..." && exit 1
```

File: scripts/two-chainz-wasmd [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  fi
+fi
+
+# Delete data from old runs
+rm -rf $WASMD_DATA &> /dev/null
+rm -rf $RELAYER_CONF &> /dev/null
+
+# Stop existing wasmd processes
+killall wasmd &> /dev/null
+killall akash &> /dev/null
```
Comment (colin-axner) [inline scripts/two-chainz-wasmd]:
```suggestion
```

I don't think this is needed?

**Review by colin-axner — COMMENTED** [review]
LGTM just left some copy/paste fixes

**Comment by orkunkl** [comment]
@colin-axner sorry for closing this PR, I applied your fixes but for some reason couldn't resolve conflicts. Going to create a new one.

**Pull request closed without merging**